packaging/deb/postinst

```diff
@@ -11,7 +11,8 @@ if [ "$1" = "configure" ]; then
 Logs:   journalctl -f -u bee.service
 Config: /etc/bee/bee.yaml
 
-Bee has SWAP enabled and by default is using slock.it goerli ethereum endpoint.
+Bee has SWAP enabled and by default is using ws://localhost:8546 ethereum endpoint.
+If needed obtain Infura account and set it.
```

Comment:
I liked your comment in `packaging/docker/README.md` (line 15). Could you update this line and give a bit more direction to the user (as in `packaging/docker`?